lib/mumukit/auth/slug.rb

```diff
@@ -114,7 +133,7 @@ def match(pattern, part)
 
     def self.validate_slug!(slug)
       unless slug =~ /\A[^\/\n]+\/[^\/\n]+\z/
```

Comment:
maybe we should use SLUG_REGEXP here now?

Reply:
This one is checking for SOMETHING/SOMETHING while the other one is checking the validity of that SOMETHING as far as I understand. Using SLUG_REGEXP as part of this validation, wouldn't make this one more restrictive and the other validations obsolete? (You are the king of regex, so correct me if I'm wrong).

I do agree that it feels quite odd that the validations are distributed between this method and the initialization.

Reply:
> I do agree that it feels quite odd that the validations are distributed between this method and the initialization.

right, that's what I was going for, it seems odd that we're doing two different regexp validations over the same thing. We could end up with cases where, for a given slug, `initialize` fails but `validate_slug!` passes... isn't that weird?

Reply:
I was thinking we could just replace the `[^\/\n]` parts in this regex with a `SLUG_REGEXP` interpolation, but I just noticed that would cause trouble because `SLUG_REGEXP` is anchored... maybe we could do something like
```ruby
SLUG_PART = /[[[:alnum:]]_ .-]+/
SLUG_REGEXP = /^#{SLUG_PART}\/#{SLUG_PART}$/
```

Reply:
I am not even sure we actually need this method :man_shrugging: